ebos/ecltransmissibility.hh

```diff
@@ -535,8 +535,10 @@ private:
 
         for (auto it = trans.begin(); it != trans.end(); ++it, ++key, ++perform)
         {
-            if(perform)
+            if(perform) {
                 fp->apply_tran(*key, *it);
+                fp->drop_tran(*key);
```

Comment:
This is run twice. First before loadbalancing and then after. Hence this will clear the  transmissibilities before the loadbalance happens and in parallel we will not see them. Also on rank 0 there are to transmissibility calculators to be cleared.

Reply:
> This is run twice. First before loadbalancing and then after.

OK - that was important information - I did not know. Would you like to suggest a better callsite or should I dig in.